core/common/src/thrift/exception.thrift

```diff
@@ -117,5 +120,5 @@ enum TExceptionStatus {
 
 exception AlluxioTException {
   2: string message
-  4: TExceptionStatus status
+  4: TStatus status
```

Comment:
can we keep some sort of indication that the 1 and 3 should be not reused? (e.g. next available ID, or commented out version of the removed fields)